lib/crawler.js

```diff
@@ -751,12 +938,11 @@ Crawler.prototype.queueURL = function(url, queueItem) {
     }
 
     // Pass this URL past fetch conditions to ensure the user thinks it's valid
-    var fetchDenied = false;
-    fetchDenied = crawler._fetchConditions.reduce(function(prev, callback) {
+    var fetchDenied = crawler._fetchConditions.reduce(function(prev, callback) {
         return prev || !callback(parsedURL);
     }, false);
 
-    if (fetchDenied) {
+    if (fetchDenied || crawler._robotsTxtFetchCondition(parsedURL)) {
```

Comment:
Well it's a "condition by which we fetch things", but it's more than a fetch condition, I felt. So I wouldn't necessarily expose it as a fetch condition, I'd expose it as a top level method similar to the other methods to parse and validate URLs.